packages/website/src/components/FinancialContributors/Sponsors/styles.module.css

```diff
@@ -29,40 +29,42 @@
   border: none;
 }
 
-.tier-sponsor {
+.tier-platinum-sponsor {
   gap: 32px 16px;
 }
 
-.tier-sponsor img {
+.tier-platinum-sponsor img {
   display: inline-block;
-  max-height: 120px;
-  max-width: 120px;
-  width: 120px;
+  max-height: 130px;
+  max-width: 130px;
+  width: 130px;
 }
 
-.tier-supporter,
-.tier-contributor {
+.tier-gold-supporter,
+.tier-silver-supporter {
   align-items: center;
 }
 
-.tier-supporter {
+.tier-gold-supporter {
   font-size: 0.95rem;
   line-height: 1;
-  gap: 24px;
-  padding-top: 16px;
+  gap: 12px;
+  padding-top: 12px;
 }
 
-.tier-supporter img {
+.tier-gold-supporter img {
   max-height: 75px;
   max-width: 75px;
   width: 75px;
 }
 
-.tier-contributor {
-  gap: 4px 24px;
+.tier-silver-supporter {
+  gap: 4px 16px;
+  margin: auto;
+  max-width: 75%;
```

Comment:
Right now we only have 9 silver supporters. Once we get up to 13 or 14 it'll look more reasonable at full width.